src/plugins/three/DebugTilesPlugin.js

```diff
@@ -648,6 +669,113 @@ export class DebugTilesPlugin {
 
 		}
 
+	}
+
+	_updateHelperMaterial( tile, material ) {
+
+		if ( tile.__visible || ! this.displayParentBounds ) {
+
+			material.linewidth = 1;
+			material.opacity = 1;
+
+		} else {
+
+			material.linewidth = 4;
+			material.opacity = 0.2;
```

Comment:
I'd prefer to avoid using line width so we're consistent between platforms - it's not working on my machine, for eample. As you say it looks like we'll need a better treatment for these lines, though, so they're more distinguishable. The random colors for each depth tier don't really help, either. But I think we can evolve this as it's actually used for debugging stuff - the logic for toggling parents is great to have added, though.

Reply:
I removed the line width change.

Reply:
On the top of my heads, here are a few ideas to help visualize the relationship between parent and child bounds:
- optionally connecting lines between the center of the parent bounds to the centers of child bounds, the result would look like some kind of "nervous system", that would help figuring the structure of the hierarchy
- using [materials that support dashes](https://threejs.org/docs/?q=dash#api/en/materials/LineDashedMaterial) and use dashed lines for intermediate bounds
- optionally, when hovering a tile (using raycasting), display only the bounds of the sub-tree that this tile belongs to, up to the root tile

Reply:
> optionally, when hovering a tile (using raycasting), display only the bounds of the sub-tree that this tile belongs to, up to the root tile

This one I think would probably be the most useful. I think the other visual treatments for the parent or child lines will be really noisy and a bit difficult to understand. I'll merge this for now and we can consider this in the future